community/daniele-borges.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
## Olá, meu nome é Daniele 
### Quem sou eu?
Moro no interior de São Paulo e tenho 18 anos. Meu interesse pela área começou durante o curso técnico integrado ao ensino médio de Desenvolvimento de Sistemas que fiz na ETEC. Após concluir o ensino médio, decidi continuar na área de TI e atualmente estou cursando ADS - Análise e Desenvolvimento de Sistemas. Tenho uma grande afinidade com o front-end, onde posso criar e dar vida às telas.

### Contatos

[![Gmail](https://img.shields.io/badge/Gmail-333333?style=for-the-badge&logo=gmail&logoColor=red)](mailto:danileborges11@gmail.com)
[![LinkedIn](https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/daniele-borges-8182071aa/)
[![Instagram](https://img.shields.io/badge/-Instagram-%23E4405F?style=for-the-badge&logo=instagram&logoColor=white)](https://www.instagram.com/danielebborges?igsh=Y29od2g0Y2duanY2)
[![GitHub](https://img.shields.io/badge/GitHub-100000?style=for-the-badge&logo=github&logoColor=white)](https://github.com/Danielebborges)



### Habilidades
![Figma](https://img.shields.io/badge/Figma-696969?style=for-the-badge&logo=figma&logoColor=figma)
![HTML](https://img.shields.io/badge/HTML-E34F26?style=for-the-badge&logo=html5&logoColor=white)
![CSS](https://img.shields.io/badge/CSS-1572B6?style=for-the-badge&logo=css3&logoColor=white)
![PHP](https://img.shields.io/badge/PHP-777BB4?style=for-the-badge&logo=php&logoColor=white)

### Aprendendo
![Python](https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)